src/components/posts/buttons/LikeButton.js

```diff
@@ -0,0 +1,21 @@
+function LikeButton({ id, setLikeCount, likeCount }) {
+  function likePost() {
+    const data = { likes: likeCount + 1 };
+    fetch(
+      `https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts/${id}.json`,
+      {
+        method: "PATCH",
+        body: JSON.stringify(data),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    ).then(() => {
+      setLikeCount(likeCount + 1);
```

Comment:
Nice way to cutdown on the number of re-renders

Reply:
Thanks